dockerfiles/db/Dockerfile

```diff
@@ -1,25 +1,36 @@
 FROM postgres:13 as build
 
-RUN apt-get update
-RUN apt-get install build-essential git cmake curl -y
-RUN apt-get install postgresql-server-dev-13 -y
-RUN apt-get install python2 -y
+RUN \
+  apt-get update && \
+  apt-get install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    cmake \
+    git \
+    postgresql-server-dev-13 \
+    python2 \
+    && \
+  rm -rf "/var/lib/apt/lists/*"
 
-RUN git clone https://github.com/graphql/libgraphqlparser.git \
-    && cd libgraphqlparser \
-    && cmake . \
-    && make install
+RUN \
+  git clone https://github.com/graphql/libgraphqlparser.git && \
+  cd libgraphqlparser && \
+  cmake . && \
+  make install
 
 COPY . pg_graphql
-WORKDIR pg_graphql
+WORKDIR /pg_graphql
 RUN make install
 
 FROM postgres:13 as main
 
+# libgraphqlparser
 ENV LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}"
-
-COPY --from=build ["/usr/local/include/graphqlparser", "/usr/local/include/graphqlparser"]
 COPY --from=build ["/usr/local/lib/libgraphqlparser.so", "/usr/local/lib/libgraphqlparser.so"]
-COPY --from=build ["/usr/lib/postgresql", "/usr/lib/postgresql"]
-COPY --from=build ["/usr/share/postgresql", "/usr/share/postgresql"]
-COPY --from=build ["/pg_graphql/pg_graphql.control", "/usr/share/postgresql/13/extension/pg_graphql.control"]
+
+# pg_graphql
```

Comment:
@olirice is the directory you are referring to `/usr/lib/postgresql/13/lib/bitcode/`?

Reply:
Unless it has a negative impact on the image (lmk) I liked
```
COPY --from=build ["/usr/lib/postgresql", "/usr/lib/postgresql"]
COPY --from=build ["/usr/share/postgresql", "/usr/share/postgresql"]
```

because it won't require changing the docker image when e.g. the version in `pg_graphql--<version>.sql` changes and should stay constant if the base image changes from postgres 13 to 14 etc

is there a benefit for making it more specific?

Reply:
I can name a few but it's arguable:

- explicitly declaring what files are needed at runtime (I wanted to know)
- the copy is faster because you only copy the files you need (I don't have data to show how much slower)

The final image size is the same because the `FROM` image is the same. I personally like to see exactly what happens, and I like to see why/what breaks and have no surprises. For example, if the build phase adds unnecessary files in `/usr/lib/postgresql` and `/usr/share/postgresql` you won't know, for example `/usr/local/include/...`.

What's your recommendation?

Reply:
thanks for the explanation

for now I'd like to keep the flexibility of copying the folder

Reply:
Understood. Just fyi these are the only files needed at runtime:

```
"/usr/local/lib/libgraphqlparser.so"
"/usr/lib/postgresql/13/lib/pg_graphql.so"
"/usr/share/postgresql/13/extension/pg_graphql--0.1.sql"
"/usr/share/postgresql/13/extension/pg_graphql.control"
```

Not sure if this changes how you feel about things. I am making the change now.